packages/components/form/src/form.stories.ts

```diff
@@ -163,6 +164,10 @@ export const All: Story = {
         <sl-text-field ?disabled=${disabled} name="textField" placeholder="Placeholder" required></sl-text-field>
       </sl-form-field>
 
+      <sl-form-field hint="Hint text" label="Date field">
+        <sl-date-field name="dateField" placeholder="Placeholder" required></sl-date-field>
```

Comment:
It looks like the validation is not working?

<img width="1028" alt="image" src="https://github.com/user-attachments/assets/6c25e52d-d0e6-4c4b-b610-b2143d90ea17" />